client/my-sites/email/email-providers-stacked-comparison/provider-cards/utils.tsx

```diff
@@ -1,9 +1,13 @@
+import formatCurrency from '@automattic/format-currency';
 import {
 	RequestCartProduct,
 	ResponseCart,
 	ShoppingCartManagerActions,
 } from '@automattic/shopping-cart';
+import { translate } from 'i18n-calypso';
 import page from 'page';
+import React, { FunctionComponent, ReactElement } from 'react';
```

Comment:
We should not be importing `React`, and we should import the other two items as types.
```suggestion
import type { FunctionComponent, ReactElement } from 'react';
```